packages/common/src/application/model.ts

```diff
@@ -22,198 +26,474 @@ const MODEL_FEATURE: Record<string, FeatureFlags> = {
   SPEECH_TO_SPEECH: { speechToSpeech: true },
   // Additional Flags
   LIGHT: { light: true },
+  LEGACY: { legacy: true },
 };
-export const modelFeatureFlags: Record<string, FeatureFlags> = {
+
+export const modelMetadata: Record<string, ModelMetadata> = {
```

Comment:
I also considered creating a separate list to manage only displayName apart from FeatureFlags, but I integrated them because there was a possibility of omissions when adding models, etc.